app/controllers/act/event_categories_controller.rb

```diff
@@ -0,0 +1,109 @@
+class Act::EventCategoriesController < ActController
+  def index
+    if !(can? :read, Act::EventCategory)
+      render 'not_allowed', locals: { :desired => "read" }
+      return
+    end
+  end
+
```

Comment:
Please use comments to document controller actions.

Reply:
jasně